build.proj

```diff
@@ -59,5 +59,8 @@
 
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
 
+  <!-- Define an empty Execute target for Arcade's build.proj to find -->
+  <Target Name="Execute" />
```

Comment:
Do you have a binlog from when you discovered this was needed?  You shouldn't have to do this, I'm curious why you had to?

Reply:
I'll generate another, basically Standard's `build.proj` was being inluded in `@(ProjectToBuild)` & was getting built with the `Execute` target by Arcade's `build.proj`.

Reply:
Ah, I see.  I believe that's https://github.com/dotnet/arcade/issues/1452, which I hope to clean up relatively soon.  sorry about that.